src/azul/service/elasticsearch_service.py

```diff
@@ -351,9 +351,9 @@ def _generate_paging_dict(self, es_response, pagination):
 
         # To return the type along with the value, return pagination variables
         # 'search_after' and 'search_before' as JSON formatted strings
-        if search_after[0] is not None:
```

Comment:
s/paginiation/pagination in commit title for this
also please open ticket and make it a freebie